src/components/BuyManaWithFiatModal/FeedbackModal.css

```
@@ -22,6 +22,10 @@
   padding: 32px 24px 34px;
 }
 
+.dcl.feedback-modal .content img {
+  content: url(../../assets/feedback-stars.svg);
+}
+
 .dcl.feedback-modal .content img.stars {
   margin-bottom: 40px;
```

```suggestion
  content: url(../../assets/feedback-stars.svg);
```
Can we add this in here so we avoid another css definition?